userspace/libsinsp/filterchecks.cpp

```diff
@@ -1312,7 +1312,15 @@ uint8_t* sinsp_filter_check_fd::extract(sinsp_evt *evt, OUT uint32_t* len, bool
 		break;
 	case TYPE_UID:
 		{
-			if(evt->get_type() == PPME_CONTAINER_JSON_E || evt->get_type() == PPME_CONTAINER_JSON_2_E)
+			// note: we need to skip events that have no thread info
```

Comment:
what about a more general check `if(m_tinfo == nullptr){return NULL;}` Since you have already done an effort to highlight the critical events we can leave them in a comment maybe, WDYT?

Reply:
It's a good point. Relying on `m_tinfo` makes us trust that it's always set properly. I have no strong preference here!